kit/brain_board/updating-the-brain.md

```diff
@@ -0,0 +1,36 @@
+---
```

Comment:
Suggestion: The updates list page is pretty small - perhaps these update instructions should go there instead? That way there's a single page for "what are the updates" and "how do I install them".

Reply:
This page will get much longer when we add info about the file based update system.